src/client/components/BaseLayout/index.jsx

```diff
@@ -40,10 +43,12 @@ const BaseLayout = ({
   children,
 }) => {
   const classes = useStyles()
+  const isIE = useIsIE()
   return (
     <>
       <CssBaseline />
       <Masthead />
+      {isIE && <BannerForIE />}
```

Comment:
We really only need to show this on the UserPage, can we move it there?

Reply:
I have added an additional check to ensure that the banner only shows in UserPage for IE users.